test/number_test.rb

```diff
@@ -106,13 +106,13 @@ def test_edge_cases
 
   def test_bignums_are_supported
```

Comment:
Good question... I'm also not clear on the use case for referencing Fixnum and Bignum in prior versions; is it necessary to provide backwards compatibility?

Looking at other libraries (in this example from simple_form):
https://github.com/plataformatec/simple_form/pull/1456/commits/0b1c43d0b8acc215b41863a9a9e4f967a72f1e44

It looks like people are just removing references to Fixnum/Bignum entirely... which would suggest the test is no longer necessary. Unless I'm missing somethign.